src/sugar.js

```diff
@@ -83,7 +88,7 @@ SVG.extend([SVG.Element, SVG.FX], {
       : (direction === 'y') ? [0, around]
       : isFinite(direction) ? [direction, direction]
       : [0, 0]
-    this.transform({flip: directionString, origin: origin}, true)
+    this.transform({flip: directionString, ox: origin[0], oy: origin[1]}, true)
```

Comment:
cant you pass just origin here?

Reply:
yes, but I was being lazy: I'd have to change all the specs :P LOL